channels/alpha

```diff
@@ -7,6 +7,8 @@ spec:
     - name: kope.io/k8s-1.5-debian-jessie-amd64-hvm-ebs-2017-01-09
       providerID: aws
       kubernetesVersion: ">=1.5.0"
+    - providerID: gce
+      name: "cos-cloud/cos-stable-56-9000-84-2"
```

Comment:
I'm not sure either, and I don't know exactly what's up, yet. 

I had just found this result and thought it was useful at least for reference. I'm planning on digging into this tonight/tomorrow AM to see how doable it is and see how it works. I'll try to update with more news soon